charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -91,6 +91,14 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 10
             failureThreshold: 5
+          readinessProbe:
```

Comment:
Do we need to update node.yaml as node service is calling metadata service on initialization?

Reply:
I tried creating a ebs-csi-node DaemonSet with a GlobalNetworkPolicy blocking metadata access, and it indeed reported as Ready, when it probably shouldn't. However, even after adding a readinessProbe in node.yaml, the node service still reported as Ready. Not sure how to fix that issue -- if it even is an issue -- so I'll defer it for now.